packages/manager/modules/pci/src/projects/project/additional-ips/order/translations/Messages_fr_FR.json

```
@@ -0,0 +1,47 @@
+  "pci_additional_ip_create": "Ajouter une nouvelle IP additionnelle",
+  "pci_additional_ip_create_description": "Exposez votre application avec une IP agnostique",
+  "pci_additional_ip_create_step_select_ip": "Sélectionnez le type d'IP",
+  "pci_additional_ip_create_step_attach_instance": "Attachez une IP à une instance ou un réseau spécifique",
+  "pci_additional_ip_create_step_attach_instance_label": "Sélectionnez une instance",
+  "pci_additional_ip_create_action_label": "Générer le bon de commande",
+  "pci_additional_ip_create_create_instance": "<a href=\"{{ url }}\">Créer une instance</a>",
+  "pci_additional_ip_create_step_summary": "Résumé",
+  "pci_additional_ip_select_network_label": "Sélectionnez un réseau/IP",
+  "pci_additional_ip_select_network_placeholder": "Sélectionnez un réseau/IP",
+
+  "pci_additional_ip_failover_ip": "IP fail-over (OVHcloud)",
+  "pci_additional_ip_failover_ip_description": "solution optimisée pour les opérations de basculement inter-produits OVHcloud. Configuration manuelle nécessaire, prise en charge des cas d'utilisation du cloud hybride. Payé d'avance.",
+  "pci_additional_ip_create_step_select_region_failover_ip": "Sélectionnez un pays pour votre nouvelle IP fail-over OVHcloud",
+  "pci_additional_ip_create_step_select_region_description_failover_ip": "Pour plus d'informations sur les IP fail-over, <a href=\"{{ guideLink }}\" target=\"_blank\">cliquez ici.</a>",
+  "pci_additional_ip_create_step_attach_instance_info_failover_ip": "Cette IP sera directement connectée à l'instance (ancien mode, sans Public Gateway). Veuillez sélectionner une instance à utiliser. Une opération manuelle sur l'instance est nécessaire, <a href=\"{{ guideLink }}\" target=\"_blank\">plus de détails ici</a>",
+  "pci_additional_ip_create_no_instance_message_failover_ip": "Aucune instance avec réseau privé n'a été détectée dans la région sélectionnée. Merci d'en créer une avant d'attacher une IP flottante.",
+
+  "pci_additional_ip_floating_ip": "IP flottantes (Public Cloud)",
+  "pci_additional_ip_floating_ip_description": "solution optimisée pour les cas d’utilisation Public Cloud, l’automatisation (Terraform) et le support de réseaux privés régionalisés. Disponible en modèle de facturation à l’usage.",
+  "pci_additional_ip_create_step_select_region_floating_ip": "Sélectionnez une région pour votre nouvelle IP flottante",
+  "pci_additional_ip_create_step_select_region_description_floating_ip": "Vous ne trouvez pas la région que vous désirez dans la liste ? Pour en savoir plus sur la disponibilité des produits par région, <a href=\"{{ guideLink }}\" target=\"_blank\">cliquez ici.</a>",
+  "pci_additional_ip_create_step_attach_instance_description_floating_ip": "Seules les instances avec des réseaux privés sont affichées",
+  "pci_additional_ip_create_no_instance_message_floating_ip": "Aucune instance avec réseau privé n'a été trouvée dans la région sélectionnée. Veuillez en créer un avant d'attacher une IP flottante.",
```

bit confusing this phrase. We have to create a private network or we have to create an instance ?
```suggestion
  "pci_additional_ip_create_no_instance_message_floating_ip": "Aucune instance avec réseau privé n'a été trouvée dans la région sélectionnée. Veuillez en créer un avant d'attacher une IP flottante.",
```

OR

```suggestion
  "pci_additional_ip_create_no_instance_message_floating_ip": "Aucune instance avec réseau privé n'a été trouvée dans la région sélectionnée. Veuillez créer une instance avant d'attacher une IP flottante.",
```

---

we have to create a instance with private network. This is the text provided by content owner and it has been reviewed.